### DISCUSSION TITLE
reset pin as input; AVR32DA28

### DISCUSSION THREAD

**Discussion by flpxra** [SpenceKonde/DxCore#328] [Q&A] [answered]
I am trying to use reset pin as input but I have a compile time error:
badArg ("analogRead called with constant pin that is not a valid analog pin");

Of course I've selected Optiboot and Reset pin function as Input. My MCU is AVR32DA28.

No matter what I call:
`analogRead(PIN_PF6);`
`analogRead(22);`

**Comment by grandaspanna** [comment, marked as answer]
My read of the datasheet is that PF6 is not available for any analog functions - input or output. At best, you can do a digitalRead, but not a digitalWrite. See page 18: https://ww1.microchip.com/downloads/aemDocuments/documents/MCU08/ProductDocuments/DataSheets/AVR32DA28-32-48-Data-Sheet-40002228B.pdf
